ush/module-setup.sh

```diff
@@ -54,10 +54,11 @@ elif [[ $MACHINE_ID = cheyenne* ]] ; then
 
 elif [[ $MACHINE_ID = container ]] ; then
     # We are in a container
-    if ( ! eval module help > /dev/null 2>&1 ) ; then
+    # if ( ! eval module help > /dev/null 2>&1 ) ; then
         source /usr/lmod/lmod/init/bash
-    fi
+    # fi
```

Comment:
Suggest adding a comment as to why the lmod init script needs to be sourced in the container.  I'm guessing on the reasoning below:

```suggestion
        # Always source the lmod init script to override the system module paths and instead use the container modules
        source /usr/lmod/lmod/init/bash
```